src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-table" context="ddl">
+        <createTable tableName="temporary_certificate">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="temporary_certificate_pkey"/>
+            </column>
+            <column name="certificate_number" type="varchar(20)">
```

Comment:
`temporary_certificate_number` felt quite long and wordy ?, but happy for suggestions

(its called `vac_certificate` in `certificate` table)

Reply:
It's in the `temporary_certificate` table, so `certificate_number` sounds good to me 👍